src/main/java/com/example/demo/repository/SavedEmails.java

```diff
@@ -0,0 +1,70 @@
+package com.example.demo.repository;
+
+import com.example.demo.consoleInfo.Email;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import org.springframework.stereotype.Component;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+import java.util.List;
+
+@Component
+public class SavedEmails implements EmailRepo {
+    private final static String fileForStorage = "listOfEmails.json";
+    private final ObjectMapper mapper;
+    public SavedEmails(ObjectMapper mapper) {
+        this.mapper = mapper;
+    }
+    @Override
+    public void save(List<Email> email) throws IOException {
```

Comment:
Generally this method shouldn't override the whole storage. You don't need to load everything to be able to add one item into storage. This component should encapsulate complexity of the storage for the dependent components. Please, make it work in append mode